barretenberg/cpp/scripts/test_civc_standalone_vks_havent_changed.sh

```diff
@@ -13,7 +13,7 @@ cd ..
 # - Generate a hash for versioning: sha256sum bb-civc-inputs.tar.gz
 # - Upload the compressed results: aws s3 cp bb-civc-inputs.tar.gz s3://aztec-ci-artifacts/protocol/bb-civc-inputs-[hash(0:8)].tar.gz
 # Note: In case of the "Test suite failed to run ... Unexpected token 'with' " error, need to run: docker pull aztecprotocol/build:3.0
-pinned_short_hash="9d8e9555"
+pinned_short_hash="f49258af"
```

Comment:
Change is due entirely to polynomial `databus_id` which is just the polynomial s.t. coeff[i] == i, for i < N. I've made N larger (for now).